sources/packages/web/src/components/institutions/profile/InstitutionProfileForm.vue

```diff
@@ -47,7 +47,7 @@ export default {
     },
     allowedRole: {
       type: String as PropType<Role>,
-      required: true,
+      required: false,
```

Comment:
Why this type of change is part of this PR?

Reply:
it is a not required Prop, 
![image](https://user-images.githubusercontent.com/77353155/212400984-f8defe3d-fa50-46a9-ac53-2651b58b2d35.png)
for me, this file was showing error
![image](https://user-images.githubusercontent.com/77353155/212401128-5e5ed9fe-779d-4e3a-b7f5-98e78d846847.png)